.flake8

```diff
@@ -4,4 +4,4 @@
 exclude = venv*,__pycache__,cache
 ignore = W503
 max-complexity = 8
-max-line-length = 120
+max-line-length = 130
```

Comment:
This brings the config in line with [notification-admin](https://github.com/cds-snc/notification-admin/blob/master/pyproject.toml#L2) and [notification-api](https://github.com/cds-snc/notification-api/blob/master/pyproject.toml#L2)